source/_integrations/tplink_omada.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: TP-Link Omada SDN Controller
+description: Instructions on integrating TP-Link Omada SDN networking devices to Home Assistant.
+ha_category:
+  - Hub
+ha_release: '2023.03'
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@MarkGodwin'
+ha_domain: tplink_omada
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
+
+The `tplink_omada` integration allows you to control your [TP-Link Omada SDN Devices](https://www.tp-link.com/us/omada-sdn/) such as network switches, access points and internet gateways. It requires a local Omada controller (software or hardware controllers), and does not currently support the paid-for TP-Link Omada Cloud Management SAAS Platform.
+
+The integration provides basic configuration and status of Omada devices controlled by the controller. All supported devices connected to the controller will be added to Home Assistant.
+
+## Configuration UI
+
+- Host: Enter the URL of the Omada management interface.
+- Verify SSL Certificates: Enable to check the validity of the certificate presented by the Omada controller.
+- Username/Password: A user account with permissions to view & configure the site is required.
+
+### Multiple Sites
+If you have multiple sites managed by your controller, you will be prompted to choose which site to manage.
```

Comment:
Move this below the config flow include (remove the "Configuration UI" header)

Reply:
Done. Thanks